source/tutorials/orchestration/terraform.rst

```diff
@@ -0,0 +1,88 @@
+.. _tutorial_terraform:
+
+###################################
+Using Terraform with Catalyst Cloud
+###################################
+
+Terraform is a cloud-agnostic infrastructure lifecycle tool commonly
+used to implement Infrastructure as Code practices. You can read
+more about Terraform on their website at http://terraform.io/.
+
+The OpenStack provider in Terraform is used to manage resources on 
+Catalyst Cloud. 
+
+.. note::
+
+    While Catalyst Cloud has tested and makes contributions to 
+    Terraform, we do not guarantee that Terraform will always
+    support our platform versions or correctly implements 
+    OpenStack support. Where we know of issues, we will note
+    those issues on the service pages in this documentation.
+
+Basic Configuration
+===================
+
+A provider block is needed to interact with Catalyst Cloud from your
+Terraform template. A simple example, which uses authentication 
+from your environment, would be:
+
+.. code-block::
+
+    terraform {
+        required_providers {
+            openstack = {
+                source = "terraform-provider-openstack/openstack"
+                version = "1.48.0"
+            }
+        }
+    }
+
+    provider "openstack" {
+        use_octavia = true
+        max_retries = 10
+    }
+
+In the example above, you may need to change the version number to
```

Comment:
Where do they find the version number to use? https://registry.terraform.io/providers/terraform-provider-openstack/openstack/latest/docs ?